README.md

```diff
@@ -74,13 +73,13 @@ But it may not validate any:
 You can run Zebra using our Docker image.
 This command will run our latest release, and sync it to the tip:
 
-<!-- TODO: replace with `docker run zfnd/zebra` when we release 1.0.0 -->
-
-```sh
-docker run zfnd/zebra:1.0.0-rc.0
+```shell
+docker run -d zfnd/zebra:1.0.0-rc.0
```

Comment:
Nitpick: usually I try to spell out options in user documentation and shared developer scripts, so people don't have to look them up.

```suggestion
docker run --detach zfnd/zebra:1.0.0-rc.0
```

Reply:
Merged. But the approval got dismissed as a result.